gulpfile.js

```diff
@@ -201,22 +213,45 @@ gulp.task('vendor-scripts', () => beginPipe('./static/vendor/**/*.js')
 	.pipe(uglify())
 	.pipe(gulp.dest(`./build/${themeName()}/vendor`)));
 
+
+// =========================
+// COPY (Symlink) Files
+// =========================
+
 // copy other vendor files
-gulp.task('vendor-assets', () => beginPipe([
+gulp.task('vendor-assets', () => beginPipeLink([
 	'./static/vendor/**/*.*',
 	'!./static/vendor/**/*.js',
 	'!./static/vendor/**/*.{css,sass,scss}',
 ]).pipe(gulp.dest(`./build/${themeName()}/vendor`)));
 
 // copy vendor-optimized files
-gulp.task('vendor-optimized-assets', () => beginPipe(['./static/vendor-optimized/**/*.*'])
+gulp.task('vendor-optimized-assets', () => beginPipeLink(['./static/vendor-optimized/**/*.*'])
 	.pipe(gulp.dest(`./build/${themeName()}/vendor-optimized`)));
 
 // copy node modules
 const nodeModules = ['mathjax', 'font-awesome'];
```

Comment:
here we should add information to node modules, which files have to be copied only. if nodeModules gets objects with name and regex's that could work. most node modules have only a build folder that is used

Reply:
done